src/qt/coincontroldialog.cpp

```diff
@@ -627,6 +627,15 @@ void CoinControlDialog::updateLabels(CCoinControl& m_coin_control, WalletModel *
     }
 }
 
+void CoinControlDialog::changeEvent(QEvent* e)
+{
+#ifdef Q_OS_MACOS
+    if (e->type() == QEvent::PaletteChange) {
+        updateView();
+    }
```

Comment:
gui 275: this has no effect for us because our gui themes do not care about os palettes/themes. I'd suggest to drop this backport completely and mark it DNM.

Reply:
@PastaPastaPasta 

> gui 375 feels maybe incomplete, but I didn't dig into it. one other nit

gui-275 is dropped